pkg/apis/metal/validation/cloudprofile_test.go

```diff
@@ -89,5 +89,181 @@ var _ = Describe("CloudProfileConfig validation", func() {
 				"Detail":   Equal("the control plane has a partition that is not a configured zone in any of the cloud profile regions: [partition-a partition-b partition-c]"),
 			}))))
 		})
+
+		It("should pass properly configured control plane partitions with netwock isolation", func() {
+			cloudProfileConfig.MetalControlPlanes = map[string]apismetal.MetalControlPlane{
+				"prod": {
+					Partitions: map[string]apismetal.Partition{
+						"partition-b": {
+							NetworkIsolation: &apismetal.NetworkIsolation{
+								AllowedNetworks: apismetal.AllowedNetworks{
+									Ingress: []string{"10.0.0.1/24"},
+									Egress:  []string{"100.0.0.1/24"},
+								},
+								DNSServers: []string{"1.1.1.1", "1.0.0.1"},
+								NTPServers: []string{"134.60.1.27", "134.60.111.110"},
+								RegistryMirrors: []apismetal.RegistryMirror{
+									{
+										Name:     "metal-stack registry",
+										Endpoint: "https://r.metal-stack.dev",
```

Comment:
Better not use the real endpoint address here, this registry is for internal-use only.

Reply:
Used a different url for mocking instead